.trybe/requirements.json

```
@@ -0,0 +1,88 @@
+{
+  "requirements": [
+    { 
+      "description": "1 - Crie uma barra verde na parte superior da página",
+      "bonus": false
+    },
+    { 
+      "description": "2 - A barra superior deve conter o logotipo do Trybewarts no canto esquerdo com a classe trybewarts-header-logo",
```

```suggestion
      "description": "2 - Adicione uma barra superior contendo o logotipo do Trybewarts no canto esquerdo com a classe trybewarts-header-logo",
```
Penso que seria interessante um comando mais direto. Analisem se faz sentido.

---

Bom, se a questão é a imperatividade do comando acho que faz mais sentido ficar assim:

```suggestion
      "description": "2 - Adicione o logotipo da Trybewarts com a classe "trybewarts-header-logo" no canto esquerdo da barra superior,
```

Digo isso porque no 1 a gente já manda a pessoa criar a barra.